src/lib/core/CHIPTLVReader.cpp

```diff
@@ -54,6 +54,14 @@ void TLVReader::Init(const uint8_t * data, size_t dataLen)
     ImplicitProfileId = kProfileIdNotSpecified;
 }
 
+CHIP_ERROR TLVReader::Init(const ByteSpan & span)
+{
+    VerifyOrReturnError(!span.empty(), CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Why is this invalid?  Reading an empty thing might be reasonable, depending on what the caller is doing....  The actual reads will fail as needed.

Reply:
updated